kubedifflib/_diff.py

```diff
@@ -151,6 +151,8 @@ def check_file(printer, path, config):
 
     differences = 0
     for data in expected:
+      if not data:
```

Comment:
```suggestion
      # data can be None, e.g. in cases where the doc ends with a '---'
      if not data:
```

Reply:
I think it'd be good to explain in which cases `data` can be `None`.

Reply:
Thanks fo the input. I've added the comment as suggested just now.